romanNumeralsDecoder/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Converts a Roman numeral to its decimal value.
+ * @param {string} roman - The Roman numeral to convert.
+ * @returns {number} - The decimal value of the Roman numeral.
+ */
+
+const romanNumeral = {
+  I: 1,
+  V: 5,
+  X: 10,
+  L: 50,
+  C: 100,
+  D: 500,
+  M: 1000,
+};
+export function solution(roman) {
+  // Array of numbers that correspond to the Roman numeral
+  const arrOfNumbers = roman.split("").map((number) => romanNumeral[number]);
+  // Perform the reduction to calculate the decimal value
+  const result = arrOfNumbers.reduce(
+    (sum, currentNumber, index, arrOfNumbers) => {
```

Comment:
The `arrOfNumbers` argument is used multiple times, even outside of the reducer. It might be better to rename argument altogether.